aiida/cmdline/commands/work.py

```diff
@@ -8,12 +8,19 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import click
+import logging
 from tabulate import tabulate
 from aiida.cmdline.commands import work, verdi
 from aiida.cmdline.baseclass import VerdiCommandWithSubcommands
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+LOG_LEVELS = [
```

Comment:
Ah no using ``_levelNames`` is a bad idea.. they changed it across Python versions.